packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -496,7 +496,8 @@ class BottomSheetSuspendedCurve extends Curve {
   }) : assert(startingPoint != null),
        assert(curve != null);
 
-  /// The progress value at which [curve] should begin.
+  /// The progress value at which [curve] should begin. Defaults to
+  /// [Curves.easeOutCubic].
```

Comment:
Nit: The pattern with documentation comments is for one sentence and then subsequent sentences should be after a new line. Something like:

```
/// The progress value at which [curve] should begin.
///
/// This defaults to [Curves.easeOutCubic].
```